### DISCUSSION TITLE
blog/yarn-classic-dependency-in-the-workspace

### DISCUSSION THREAD

**Discussion by giscus** [yejineee/yejineee.github.io#3] [General]
# blog/yarn-classic-dependency-in-the-workspace

yarn classic으로 workspace 만들고, 디펜던시 관리 방법 알아보기

http://localhost:3000/blog/yarn-classic-dependency-in-the-workspace

**Comment by minsoo-web** [comment]
yarn classic 은 안 사용한지 오래되어 잘 몰랐는데 유령 의존성 문제가 실감이 되네요. 좋은 글 감사합니다!
